src/app/services/ninja-service.ts

```diff
@@ -69,4 +69,21 @@ export class NinjaService {
     }
     return predictions;
   }
+
+  getOperations(pathwayPrediction: PathwayPredictionResult): DeCaF.Operation[] {
```

Comment:
Good to move this here for separation of concerns, to keep the components/other code calling this slimmer :+1:

Reply:
Thanks, it was Alba's advice :)